### PR TITLE
release(v0.3.1): prepare release

### PR DESCRIPTION
**Pull request by rsmitty**
This is the official v0.3.1 release.

Signed-off-by: Spencer Smith <robertspencersmith@gmail.com>

### CODE REVIEW

**Review by andrewrynhard — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,313 @@
...
+### IPMI Discovery and Automatic Setup
+
+Sidero agent by default discovers BMC (IPMI) endpoint when it first runs on the server and provisions user for Sidero automatically.
+This feature enables completely automated server and IPMI discovery on the agent boot.
+Variable `SIDERO_CONTROLLER_MANAGER_AUTO_BMC_SETUP` can be used to disable this feature during install or upgrade of Sidero.
+
+Additionally Sidero supports storing IMPI credentials in Kubernetes secrets referencing them from the Server object.
+
+**
+Note that this functionality is known to have issues with iDRAC currently. This can be worked around by updating the `<server-uuid>-bmc` secret that is created in the `sidero-system` namespace with credentials of an existing iDRAC admin user or by disabling the feature altogether with the aforementioned environment variable.
```
Comment (smira) [inline CHANGELOG.md:58]: @rsmitty probably we should remove this and regen the release?

Comment (rsmitty) [reply CHANGELOG.md:58]: oh snap, good catch. thx.

Comment (rsmitty) [reply CHANGELOG.md:366]: kk, fixed.

**Comment by smira** [comment]
@rsmitty now 0.3.1 is twice in the `CHANGELOG.md`, you might want to do `git checkout relase-0.3 CHANGELOG.md` before generating it

**Review by smira — APPROVED** [review]

**Pull request merged**